templates/named.conf.erb

```diff
@@ -21,6 +21,9 @@ options {
 <%-   end -%>
 	};
 <%- end -%>
+<%- if @forward != '' -%>
```

Comment:
I found this which suggests that @variable != '' isn't even needed, but would definitely need testing.  http://grokbase.com/t/gg/puppet-users/147by1key3/checking-a-variable-is-not-undef